spec/features/tasks/new_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'Create a task', type: :feature do
+  include_context 'when user signed in'
+
+  let(:project) { create(:project, name: 'A test project') }
+
+  scenario 'user creates a task' do
+    visit projects_path
+    click_on 'New Project'
+
+    fill_in 'Name', with: 'A test project'
+    fill_in 'Description', with: 'Test description'
+
+    click_button 'Create Project'
+
+    visit tasks_path
+    click_on 'New Task'
+
+    fill_in 'Title', with: 'A test task'
+    fill_in 'Description', with: 'Test description'
+    fill_in 'Deadline', with: 'November 27, 2021 23:59'
```

Comment:
How can you create a task without selecting a project?

Reply:
The answer is nohow. You should create a project in `before` block and then choose it using `select`

Reply:
> 

Got cha. If you create a project, it is selected by default in the field, so you can create a task without selecting a project.